compiler/src/iree/compiler/Utils/unittests/UtilsTest.cpp

```diff
@@ -6,8 +6,11 @@
 
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
+#include <thread>
```

Comment:
I think this is no longer needed

Reply:
Look, I expanded the test -- it is actually creating multiple threads now. I tested locally with TSan, that this makes the test sensitive to omitting the mutex locking in the implementation of `withGlobal`.